client/src/components/seo/index.tsx

```diff
@@ -100,7 +100,7 @@ SEO.displayName = 'SEO';
 
 export default SEO;
 
-export const query = graphql`
```

Comment:
To make it a little more obvious that this isn't a page query, could you inline it?

Reply:
sure thing